packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/RecentSubjectsCarousel.js

```diff
@@ -0,0 +1,39 @@
+import counterpart from 'counterpart'
+import { arrayOf, shape, string } from 'prop-types'
+import React from 'react'
+import { Carousel } from 'grommet'
+import SubjectThumbnail from './components/SubjectThumbnail'
+
+// TODO: Use the subject viewers from the classifier
+function RecentSubjectsCarousel (props) {
+  const { className, href, subjects } = props
```

Comment:
`className` isn't being used here. It should maybe be applied to the `Carousel` since that's the parent?